package.json

```diff
@@ -34,38 +34,57 @@
   },
   "dependencies": {
     "@babel/cli": "^7.16.0",
-    "@babel/core": "^7.16.0",
+    "@babel/core": "^7.16.5",
```

Comment:
This package will contain all dependencies to run its binaries. This way the consumer only needs the following dependencies:

```json
"dependencies": {
  "prop-types": "^15.7.2",
  "react": "^17.0.2",
  "react-dom": "^17.0.2",
  "styled-components": "^5.3.3"
},
"devDependencies": {
  "create-react-styleguide": "^7.1.0",
  "husky": "^4.3.8",
  "react-test-renderer": "^17.0.2"
},
```